compiler/src/test-integration/java/io/neow3j/compiler/InstanceInheritanceIntegrationTest.java

```diff
@@ -0,0 +1,211 @@
+package io.neow3j.compiler;
+
+import io.neow3j.devpack.ByteString;
+import io.neow3j.devpack.Hash160;
+import io.neow3j.devpack.annotations.Struct;
+import io.neow3j.devpack.events.Event2Args;
+import io.neow3j.protocol.core.response.InvocationResult;
+import io.neow3j.protocol.core.response.NeoApplicationLog;
+import io.neow3j.protocol.core.stackitem.StackItem;
+import io.neow3j.types.Hash256;
+import io.neow3j.types.NeoVMStateType;
+import io.neow3j.types.StackItemType;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+
+import java.io.IOException;
+import java.util.List;
+
+import static io.neow3j.types.ContractParameter.array;
+import static io.neow3j.types.ContractParameter.hash160;
+import static io.neow3j.types.ContractParameter.integer;
+import static io.neow3j.types.ContractParameter.string;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+public class InstanceInheritanceIntegrationTest {
```

Comment:
Add a test that has a try-catch in the constructor.

Reply:
I'll add a test.